### PR TITLE
#21 create search box for page 10 -Emily

### PR DESCRIPTION
**Pull request by EABarton123**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -2,11 +2,15 @@ import { Route, Routes } from "react-router-dom";
...
 import About from "./pages/About";
 import Home from "./pages/Home";
+import AdminJobs from "./pages/AdminJobs";
+import GradJobs from "./pages/GradJobs";
 
 const App = () => (
 	<Routes>
 		<Route path="/" element={<Home />} />
 		<Route path="/about/this/site" element={<About />} />
+		<Route path="/admin/logged/in" element={<AdminJobs />} />
```
Comment (Rafsanjani) [inline client/src/App.js:12]: Your paths are too complex. Use simple paths. something like `/about`, `/admin`

File: searchjobs.patch
Comment (Rafsanjani) [inline searchjobs.patch:1]: Don't commit this file

**Review by Rafsanjani — CHANGES_REQUESTED** [review]

**Comment by EABarton123** [comment]
made changes suggested

File: client/src/pages/AdminJobs.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import SearchJobs from "./SearchJobs";
+
+function AdminJobs() {
+	return (
+		<div>
```
[the remark this answers is not in the document]
Comment (EABarton123) [reply client/src/pages/AdminJobs.js:5]: removed the div

File: client/src/pages/GradJobs.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import GradAppliedJobs from "./GradAppliedJobs";
+import SearchJobs from "./SearchJobs";
+import "./GradJobs.css";
+
+function GradJobs() {
+	return (
+		<div>
```
Comment (Rafsanjani) [inline client/src/pages/GradJobs.js:7]: Remove redundant div here

**Review by Rafsanjani — APPROVED** [review]

File: client/src/pages/GradJobs.js (hunk 3 above)
Comment (EABarton123) [reply client/src/pages/GradJobs.js]: done

**Comment by Rafsanjani** [comment]
Looks good.

**Pull request merged**